Bluepill-cli/Bluepill-cli/Simulator/SimulatorMonitor.m

```diff
@@ -202,14 +202,15 @@ - (void)onOutputReceived:(NSString *)output {
                 [BPUtils printInfo:CRASH withString:@"%@/%@ crashed app.",
                  (self.currentClassName ?: self.previousClassName),
                  (self.currentTestName ?: self.previousTestName)];
+                self.exitStatus = BPExitStatusAppCrashed;
             } else {
                 assert(__self.testsState == Idle);
                 [BPUtils printInfo:CRASH withString:@"App crashed before tests started."];
+                self.exitStatus = BPExitStatusAppTerminatedBeforeTestStarted;
```

Comment:
we need to update the LOG information above as well?

Do we handle [BPExitStatusAppTerminatedBeforeTestStarted] and [BPExitStatusAppCrashed] differently or just log different messages?

Reply:
For BPExitStatusAppTerminatedBeforeTestStarted, if retry pass, we call it pass. For BPExitStatusAppCrashed, if retry pass, we still return appCrash.

Reply:
Got it, I think it makes more sense to name it something relevant to app so that we know it is a flakiness in the simulator/test runner.